schemas/maas-backend/customers/verification/status/response.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/customers/verification/status/response.json",
+  "description": "MaaS customer verification status",
+  "type": "object",
+  "properties": {
+    "customer": {
+      "type": "object",
+      "properties": {
```

Comment:
Also move some nested definitions to file level definitions for readability

Reply:
Done.